apps/workflow-server/gcc-workflow-server/src/main/java/com/coremedia/labs/translation/gcc/workflow/SendToGlobalLinkAction.java

```diff
@@ -177,7 +187,8 @@ void doExecuteGlobalLinkAction(Parameters params, Consumer<? super String> resul
     Locale firstMasterLocale = findFirstMasterLocale(masterContentObjects, localeMapper)
             .orElseThrow(() -> new IllegalStateException("Unable to identify master locale."));
 
-    String submissionId = submitSubmission(facade, params.subject, params.comment, firstMasterLocale, translationItemsByLocale, params.dueDate, params.workflow, params.submitter);
+    String submissionId = submitSubmission(facade, params.subject, params.comment, firstMasterLocale,
+            translationItemsByLocale, params.dueDate, params.workflow, params.submitter.getName());
```

Comment:
:+1: for moving Content Server call User#getName from LongAction's extractParameters to execute!

however, an NPE may occur now if params.submitter is null

Reply:
Oops, right. Changed.